release-notes/kvm/v11.1.0.md

```diff
@@ -1,5 +1,7 @@
 ## :zap: Giant Swarm Release 11.1.0 for KVM is now active for you! :zap:
 
+This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also added metrics Service for prometheus-operator support. This enables scraping of metrics and better monitoring on the app.
```

Comment:
```suggestion
This release upgrades the NGINX Ingress Controller app to upstream v0.28.0. It contains several non-breaking changes that fix a few issues. We also enabled the metrics service. This allows prometheus-operator to scrape for metrics and improves monitoring of the app.
--


.
```

Reply:
@cokiengchiara will commit suggestion for most of the changes are improvement, with exception of "enabled" part -  we haven't actually enabled the metrics Service by default, we supported it, it has to be enabled on-demand.